neurolang/relational_algebra_provenance.py

```diff
@@ -649,8 +721,22 @@ def selection_rap_eq_columnint(self, selection):
         )
         return self.walk(Selection(selection.relation, new_formula))
 
+    @add_match(DisjointProjection(ProvenanceAlgebraSet, ...))
```

Comment:
This is not really great, you make the parser work, but the two projections shouldn't have different semantics in the general case. Worst case both projections inherit from the `Projection` case and are processed like Projection by inheritance. But like this you are inventing general semantics that don't exist